pallets/ajuna-tournament/src/impls/fee_handler.rs

```diff
@@ -1,19 +1,18 @@
 use super::*;
-use ajuna_primitives::fee_handler::{DistributeFee, Payment};
+use ajuna_primitives::payment_handler::{DistributeFee, PaymentFee};
 use sp_arithmetic::traits::{CheckedDiv, Saturating};
-use sp_std::vec;
 
 impl<T: Config<I>, I: 'static> DistributeFee for Pallet<T, I> {
 	type AccountId = AccountIdFor<T>;
 	type Balance = BalanceOf<T, I>;
 	type FeeIdentifier = TournamentCategoryIdFor<T, I>;
-	type MaxDistributions = ConstU32<1>;
+	type FeeDistribution = PaymentFee<Self::AccountId, Self::Balance>;
```

Comment:
PaymentFee doesn't sound meaningful in English to me.  It would rather be `FeePayment`. However, because the struct itself is merely comprised of a beneficiary and an account, it think the generic name `Payment` is still fine. What was your reasoning behind changing the name?

Reply:
The name change came from the idea that it doesn't represent a number of distribution but more the concept of a fee distribution right now.